backend/spec/services/google_scraper_spec.rb

```diff
@@ -5,12 +5,32 @@
   
   let!(:user_1)         { FactoryBot.create(:user) }
 
-
   describe '#call' do
     it 'create keyword from word in csv' do
+      response = File.read(fixture_file_path('stubs/cat.txt'))
+
+      stub_request(:get, "https://www.google.com/search?hl=en&q=cat").
```

Comment:
Using Webmock directly is a bit cumbersome. That is why using VCR is usually preferred as the DSL is nicer to work with 😁